src/app/content/highlights/components/cardStyles.ts

```diff
@@ -0,0 +1,205 @@
+ * that has a selector to get the relevant stuff.
+ */
+
+const additionalWidthForCard = (cardWidth + cardContentMargin + cardMinWindowMargin) * 2;
+const minimalWidth = contentTextWidth + additionalWidthForCard;
+export const minimalWidthForCards = '(max-width: ' + remsToEms(minimalWidth) + 'em)';
+export const minimalWidthForCardsWithToc = '(max-width: ' +
+  remsToEms(minimalWidth + sidebarDesktopWidth) + 'em)';
+export const minimalWidthForCardsWithSearchResults = '(max-width: ' +
+  remsToEms(minimalWidth + searchResultsBarDesktopWidth) + 'em)';
+
+const overlapDisplay = css`
+  ${(props: CardProps) => !!props.isFocused && css`
+    left: unset;
+    right: ${cardMinWindowMargin}rem;
+    top: ${() => {
+      return getHighlightBottomOffset(props.container, props.highlight) || 0;
+    }}px;
+  `}
+  ${(props: CardProps) => !props.isFocused && css`
+    display: none;
+  `}
+`;
+
+const rightSideDisplay = css`
+  left: calc(100% - ((100% - ${contentTextWidth}rem) / 2) + ${cardContentMargin}rem);
+  right: unset;
+  top: ${(props: CardProps) => `${props.topOffset || getHighlightBottomOffset(props.container, props.highlight)}px;`}
+  ${(props: CardProps) => !!props.isFocused && css`
+    left: calc(100% - ((100% - ${contentTextWidth}rem) / 2) + ${cardFocusedContentMargin}rem);
+  `}
+`;
+
+const mobileDisplay = css`
+  ${(props: CardProps) => !!props.isFocused && css`
+    left: 0;
+    right: 0;
+    bottom: 0;
+    top: unset;
+    position: fixed;
+    padding: 0;
+  `}
+  ${(props: CardProps) => !props.isFocused && css`
+    display: none;
+  `}
+`;
+
+const fadeIn = keyframes`
+  0% {
+    opacity: 0;
+  }
+
+  100% {
+    opacity: 1;
+  }
+`;
+
+const fadeInAnimation = css`
+  animation: ${600}ms ${fadeIn} ease-out;
+`;
+
+export const mainCardStyles = css`
+  ${fadeInAnimation}
+  position: absolute;
+  padding: ${cardPadding}rem;
+  ${cardBorder}
+  ${rightSideDisplay}
+  ${disablePrint}
+
+  z-index: ${(props: CardProps) => props.zIndex};
+  transition: all 0.3s;
+
+  ${DropdownList} {
+    z-index: 1;
+  }
+
+  ${(props: {data: HighlightData}) => {
+    const data = props.data;
+
+    if (!data || !data.color) {
+      return null;
+    }
+
+    const style = highlightStyles.find((search) => search.label === props.data.color);
+
+    if (!style) {
+      return null;
+    }
+
+    return css`
+      ::before {
+        content: ' ';
+        border-radius: 0.4rem 0 0 0.4rem;
+        position: absolute;
+        top: 0;
+        left: 0
+        bottom: 0;
+        width: ${cardPadding / 2}rem;
+        background-color: ${style.focused};
+      }
+      ${theme.breakpoints.mobile(css`
+        ::before {
+          border-radius: 0.4rem 0.4rem 0 0;
+          right: 0;
+          bottom: unset;
+          width: unset;
+          height: ${cardPadding / 2}rem;
+        }
+     `)}
+    `;
+  }}
+
+  /* for some reason isTocOpen prop will not refresh after first page load and is null */
```

Comment:
tocOpen === null indicates that the state hasn't been set by the user, which means its open on desktop and closed on mobile.